template.rb

```diff
@@ -0,0 +1,78 @@
+gem "godmin", "~> 0.12.3"
```

Comment:
Perhaps we should remove the version? Bothersome to update it for every release.